### PR TITLE
bump version for utility-connect 2.0

### PR DESCRIPTION
**Pull request by xelzi**
Follow up: https://github.com/ArcadiaPower/utility-connect/pull/47

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -181,17 +181,23 @@ Data is expected in the following format for the `update` scope:
 
 Callback functions triggered at key points in the Utility Connect flow. Expects an object with key/value pairs where keys are as documented below.
 
-**`onEmit`**: callback function that is triggered when utility credentials are either created or updated in the API.
+**`onClose`**: callback function that is triggered when the Utility Connect component is closed.
+
+**`onError`**: callback function that is triggered when an error occurs within the application.
```
Comment (jakedouglas) [inline README.md:186]: Maybe something like:

```suggestion
**`onError`**: callback function that is triggered when an error occurs within the Utility Connect process.
```

File: README.md [hunk 2]
```diff
@@ -181,17 +181,23 @@ Data is expected in the following format for the `update` scope:
...
 
 **`onOpen`**: callback function that is triggered when the Utility Connect component is opened.
 
-**`onClose`**: callback function that is triggered when the Utility Connect component is closed.
+**`onSuccess`**: callback function that is triggered when utility credentials are successfully verified.
+
+**`onTimeout`**: callback function that is triggered when the Utility Connect component timesout while trying to validate utility credentials.
```
Comment (jakedouglas) [inline README.md:192]:
```suggestion
**`onTimeout`**: callback function that is triggered when the Utility Connect component times out awaiting verification of the utility credentials.
```

File: README.md [hunk 3]
```diff
@@ -181,17 +181,23 @@ Data is expected in the following format for the `update` scope:
...
 
 ```javascript
 {
-  onEmit: ({ error, data }) => { ... },
+  onClose: () => { ... },
+  onError: () => { ... },
```
Comment (jakedouglas) [inline README.md:197]:
```suggestion
  onError: ({ error }) => { ... },
```

**Comment by johnnyman727** [comment]
@xelzi It doesn't matter that much, but I think we ideally want to wait for https://github.com/ArcadiaPower/utility-connect-react/pull/10 to get merged first, just FYI.

File: README.md (hunk 2 above)
Comment (dlopuch) [inline README.md:192]: These should contain method signatures and payload specifications, eg

```
**`onSuccess(credentialsCreatedResponse)`** callback function that is triggered...

`credentialsCreatedResponse` is an object like:

Key | Type | Desc
--- | --- | ---
`data` | Object | Credentials-created API response
`data.user_id` | Number | User ID of created member
`data.whatever` | Blob | Zippity doo bop
```

File: package.json [hunk 4]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@arcadia-eng/utility-connect-react",
-  "version": "0.0.1-alpha.9",
+  "version": "0.0.2-alpha.1",
```
Comment (dlopuch) [inline package.json:3]: Major version should match utility-connect main version

**Review by dlopuch — COMMENTED** [review]
Could use some more helpful documentation

File: package.json (hunk 4 above)
Comment (xelzi) [reply package.json:3]: Yeahhhh this has been a point of contention today -- Are we really ready to version this as version 2.0.0 when the features have not been locked down? Still feels like alpha to me

Comment (xelzi) [reply package.json:3]: but went ahead and bumped -- YOLO

**Pull request merged**